teuthology/task/buildpackages/common.sh

```diff
@@ -41,9 +41,6 @@ function get_ceph() {
 
     test -d ceph || git clone ${git_ceph_url} ceph
     cd ceph
-    if test -d src ; then # so we don't try to fetch when using a fixture
-       git fetch --tags http://github.com/ceph/ceph
-    fi
```

Comment:
I'm pretty sure you still want that to ensure you have all the tags found in http://github.com/ceph/ceph, in addition to the tags that are fetched by the above clone. This is in case the clone misses a few tags for some reason.